OpenRA.Mods.Common/Widgets/Logic/MusicPlayerLogic.cs

```diff
@@ -41,12 +41,12 @@ public MusicPlayerLogic(Widget widget, Ruleset modRules, World world, Action onE
 			var playButton = panel.Get<ButtonWidget>("BUTTON_PLAY");
 			playButton.OnClick = Play;
 			playButton.IsDisabled = noMusic;
-			playButton.IsVisible = () => !Sound.MusicPlaying;
+			playButton.IsVisible = () => { !Sound.MusicPlaying && currentSong != null; };
```

Comment:
Could someone tell me how to write this right? Travis/Appveyor both throw a syntax error here but my MonoDevelop compiles it - I guess it's a mono bug and I don't want to feel more stupid than I am and do x amount of commits to guess out the proper syntax.